dotCMS/src/test/java/com/dotmarketing/util/FileUtilTest.java

```diff
@@ -0,0 +1,49 @@
+package com.dotmarketing.util;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.dotcms.UnitTestBase;
+import java.io.File;
+import java.nio.file.Paths;
+import java.util.Optional;
+import org.junit.Test;
+
+public class FileUtilTest extends UnitTestBase {
+
+    /**
+     * Given scenario: I have a file that belongs under my realAssets path (somewhere)
+     * Expected Result: The method gets me the relative part of the file's absolute path that is under the real asset path.
+     */
+    @Test
+    public void Test_Get_Relative_Path_Expect_Match(){
+        final String relativePart = "/4/1/419bacda-1e1d-412e-b73e-db514289b71f/fileAsset/test_image11600880647045.jpg";
+        final String realAssetPath = "/Users/fabrizzio/code/servers/server1/assets/";
```

Comment:
why don't you get the real asset path from the config properties?

Reply:
it's a unit test.  I wanted to avoid using api calls such as 
```
APILocator.getFileAssetAPI().getRealAssetsRootPath()
or 
Config.getStringProperty("ASSET_REAL_PATH")
```
besides, it's an example.

Reply:
Yeah, I am not sure we need the two extra methods in the FileUtils.

We also have the `ConfigUtils.getAbsoluteAssetsRootPath()` which I think has the logic and is where we should read the `/asset` path from

Reply:
Ok, I'm switching to `ConfigUtils.getAbsoluteAssetsRootPath()` but I still need a method that figures out the relative part of a file when all that I have is the absolute path of the binary. Where would you like to see that method live ? 
I thought FileUtil was a fine spot for that @wezell

Reply:
alright I'm moving these methods